plugins/vtd/bin/vtd

```diff
@@ -1,61 +1,235 @@
+
+    for file in ["bin", "Develop", "Runtime"]:
+        src = vtd_root / file
+        dst = runtime_dir / file
+        logging.debug("Create symlink: {} -> {}".format(dst, src))
+        os.symlink(src, dst)
+
+    for file in ["Distros", "Projects", "Setups"]:
+        srcdir = vtd_root / "Data" / file
+        dstdir = runtime_dir / "Data" / file
+        logging.debug("Create directory: {}".format(dstdir))
+        os.makedirs(dstdir)
+
+        for file in os.listdir(srcdir):
+            srcfile = srcdir / file
+            dstfile = dstdir / file
+            logging.debug("Create symlink: {} -> {}".format(dstfile, srcfile))
+            os.symlink(srcfile, dstfile)
+
+
+# _________________________________________________________________________
+# Command: teardown
+@main.command("teardown")
+@click.pass_obj
+def teardown(opt):
+    """Tear down an isolated VTD runtime mirror."""
+
+    logging.info("Tear down runtime mirror: {}".format(opt.runtime_dir))
+    teardown_runtime(opt.runtime_dir)
+
+
+def teardown_runtime(runtime_dir: Path) -> None:
+    if runtime_dir.exists():
+        logging.debug("Remove runtime mirror: {}".format(runtime_dir))
+        shutil.rmtree(runtime_dir, ignore_errors=False)
+
+
+# _________________________________________________________________________
+# Command: start [--skip-setup]
+@main.command("start")
+@click.option(
+    "--skip-setup",
+    is_flag=True,
+    help="Skip (re-)creation of runtime mirror.",
+)
+@click.option(
+    "-s",
+    "--vtd-setup",
+    type=click.Path(exists=True, file_okay=True, dir_okay=True),
+    help="VTD setup to use, default to currently symlinked.",
+)
+@click.option(
+    "-p",
+    "--vtd-project",
+    type=click.Path(exists=True, file_okay=True, dir_okay=True),
+    help="VTD project to use, default to currently symlinked.",
+)
+@click.option(
+    "-a",
+    "--vtd-args",
+    type=str,
+    default="",
+    help="Extra arguments to pass to VTD.",
+)
+@click.pass_obj
+def start(
+    opt,
+    skip_setup: bool,
+    vtd_setup: str,
+    vtd_project: str,
+    vtd_args: str,
+):
+    """Start a new isolated VTD simulation."""
+
+    if not skip_setup:
+        logging.info("Set up runtime mirror: {}".format(opt.runtime_dir))
+        teardown_runtime(opt.runtime_dir)
+        setup_runtime(opt.root_dir, opt.runtime_dir)
+
+    if opt.has_session():
+        raise click.UsageError(
+            "tmux session already exists: {}".format(opt.session_name)
+        )
+
+    session = opt.new_session()
+    window = session.attached_window
+    window.rename_window("vtd")
+    pane = window.attached_pane
+    pane.clear()
+
+    if vtd_setup is not None:
+        logging.info("Select setup: {}".format(vtd_setup))
+        if Path(vtd_setup).exists():
+            vtd_setup = Path(vtd_setup).resolve()
+        dst = opt.runtime_dir / "Data" / "Setups" / "Current"
+        if dst.exists():
+            os.remove(dst)
+        os.symlink(vtd_setup, dst)
```

Comment:
For the smoketest with OSI sensor model, there is an issue due to the relative symlinks from the symlinked `vtd_setup` directory. Copying `vtd_setup` would solve that problem, but is undesirable for performance reasons..